sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -263,14 +263,14 @@ public static boolean hasCrashedInLastSession() {
     /**
      * Provides information about any available crash report from the last session, if it crashed.
      * This method is a synchronous call and blocks caller thread.
-     * Use {@link #getLastSessionCrashReportAsync(ResultCallback)} for asynchronous call.
+     * Use {@link #getLastSessionCrashReport(ResultCallback)} for asynchronous call.
      *
      * @return The crash report from the last session if one was set.
-     * @see #getLastSessionCrashReportAsync(ResultCallback)
+     * @see #getLastSessionCrashReport(ResultCallback)
      */
     @Nullable
     @WorkerThread
-    public static ErrorReport getLastSessionCrashReport() {
```

Comment:
Instead of this, can you call `getInstance().getInstanceLastSessionCrashReport()` directly from Xamarin?